src/server/routes/api.ts

```diff
@@ -40,12 +40,12 @@ if (process.env.MONGODB_HOST && process.env.MONGODB_PORT && process.env.APP_ENV)
         ], { region: 'us-east-1' }
     );
 
-    database.url = 'mongodb://' + results.Parameters[1]['Value'] + ':'
-        + results.Parameters[0]['Value']
+    database.url = 'mongodb://' + results.Parameters[0]['Value'] + ':'
+        + results.Parameters[1]['Value']
         + '@' + process.env.MONGODB_HOST + ':' + process.env.MONGODB_PORT + '/agora'
         + '?authSource=admin';
 } else {
-    database.url = 'mongodb://localhost:27017/agora';
+    database.url = 'mongodb://admin:[password]@dbcluster-al7isb6nrny7.cluster-cuyxs1otyejf.us-east-1.docdb.amazonaws.com:27017/agora?authSource=admin'
```

Comment:
i won't be deploying this locally.  i will deploying it on AWS so it won't ever go here.

Reply:
I know. This is for my local